Makefile

```diff
@@ -80,6 +80,8 @@ FIREFOX_PACKAGE_FILES= \
 	manifest.json \
 	manifest.json.license \
 	icons/linux-entra-sso.svg \
+	icons/profile-outline_48.png \
+	icons/profile-outline_48.png.license \
 	popup/profile-outline.svg
```

Comment:
Maybe worth to introduce `COMMON_PACKAGE_FILES` first...

Reply:
I just split the commit into the refactoring part and the fix. For now, we just move the common part up to the common input files as this is static data. Anyways, I'm thinking of just including all static data in both firefox and chrome to make the Makefile (and testing) simpler. The added complexity is probably not worth it for saving a couple of kb. Anyways, that's something for a follow up MR.